src/Microsoft.AspNetCore.Authentication.Cookies/__TemporarySources__/TaskCacheOfT.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
This is a workaround for 'contentFiles' not yet working in MSBuild. We will be deleting these in a few weeks when we get an upgrade to NuGet that fixes the issue. cref https://github.com/aspnet/Common/issues/180